src/phoenix/server/email/sender.py

```diff
@@ -18,22 +23,44 @@ def __init__(
         username: str,
         password: str,
         sender_email: str,
-        connection_method: Literal["STARTTLS", "SSL", "PLAIN"] = "STARTTLS",
+        connection_method: ConnectionMethod = "STARTTLS",
         validate_certs: bool = True,
     ) -> None:
         self.smtp_server = smtp_server
         self.smtp_port = smtp_port
         self.username = username
         self.password = password
         self.sender_email = sender_email
-        self.connection_method = connection_method.upper()
+        self.connection_method: ConnectionMethod = connection_method
         self.validate_certs = validate_certs
 
         self.env = Environment(
             loader=FileSystemLoader(EMAIL_TEMPLATE_FOLDER),
             autoescape=select_autoescape(["html", "xml"]),
         )
 
+    async def send_welcome_email(
+        self,
+        email: str,
+        name: str,
+    ) -> None:
+        subject = "[Phoenix] Welcome to Phoenix"
```

Comment:
```suggestion
        subject = "[Phoenix] Welcome to Arize Phoenix"
```
Maybe just because people will search their inbox